nautobot/users/models.py

```diff
@@ -29,6 +28,14 @@
 #
 
 
+class UserManager(BaseManager, UserManager_):
+    """
+    Natural-key subclass of Django's UserManager.
+
+    Note that this is *NOT* based around RestrictedQuerySet.
```

Comment:
Not sure why this is the case, but changing it caused some RackReservation test cases to fail due to newly appearing User access constraints.